files/image_config/procdockerstats/procdockerstats

```diff
@@ -0,0 +1,214 @@
+            return None
+        else:
+            return stdout
+
+    def format_docker_cmd_output(self, cmdout):
+        lines = re.split("\n", cmdout)
+        keys = re.split("   +", lines[0])
+        docker_data = dict()
+        docker_data_list = []
+        for item in lines[1:]:
+            values1 = re.split("   +", item)
+            docker_data =  dict(zip(keys, values1))
+            docker_data_list.append(docker_data)
+        formatted_dict = self.create_docker_dict(docker_data_list)
+        return formatted_dict
+
+    def format_process_cmd_output(self, cmdout):
+        lines = re.split("\n", cmdout)
+        keys = re.split(" +", lines[0])
+        keylist = list(filter(None, keys))
+        process_data = dict()
+        process_data_list = []
+        for item in lines[1:]:
+            values1 = re.split(" +", str(item))
+            # To remove extra space before UID
+            val = list(filter(None, values1))
+            # Merging extra columns created due to space in cmd ouput
+            val[8:] = [''.join(val[8:])] 
+            process_data = dict(zip(keylist, val))
+            process_data_list.append(process_data)
+        return process_data_list
+
+    def convert_to_bytes(self, value):
+        unit_value = re.search('[a-zA-Z]+', value)
+        value_to_convert = float(filter(str.isdigit, value))
+        unit = unit_value.group(0)
+        UNITS_B = 'B'
+        UNITS_KB = 'KB'
+        UNITS_MB = 'MB'
+        UNITS_MiB = 'MiB'
+        UNITS_GiB = 'GiB'
+        if unit.lower() == UNITS_B.lower():
+            return int(round(value_to_convert))
+        elif unit.lower() == UNITS_KB.lower():
+            value_converted = value_to_convert * 1000
+            return int(round(value_converted))
+        elif unit.lower() == UNITS_MB.lower():
+            value_converted = value_to_convert * 1000 * 1000
+            return int(round(value_converted))
+        elif unit.lower() == UNITS_MiB.lower():
+            value_converted = value_to_convert * 1024 * 1024
+            return int(round(value_converted))
+        elif unit.lower() == UNITS_GiB.lower():
+            value_converted = value_to_convert * 1024 * 1024 * 1024
+            return int(round(value_converted))
+
+    def create_docker_dict(self, dict_list):
+        dockerdict = {}
+        for row in dict_list[0:]:
+            cid = row.get('CONTAINER ID')
+            if cid:
+                key = 'DOCKER_STATS|' + str(cid) 
+                dockerdict[key] = {}
+                dockerdict[key]['NAME'] = row.get('NAME')
+                
+                splitcol = row.get('CPU %')
+                cpu = re.split("%", str(splitcol))
+                dockerdict[key]['CPU%'] = str(cpu[0])
+
+                splitcol = row.get('MEM USAGE / LIMIT')
+                memuse = re.split(" / ", str(splitcol))
+                # converting MiB and GiB to bytes
+                dockerdict[key]['MEM'] = str(self.convert_to_bytes(memuse[0]))
```

Comment:
The values which are now all in bytes should have the units appended to the key name (`MEM_BYTES`, `MEM_LIMIT_BYTES`, `NET_IN_BYTES`, `NET_OUT_BYTES`, etc) as also requested in the design doc: https://github.com/Azure/SONiC/pull/465/files#r338856709